bintray/services/upload.go

```diff
@@ -44,13 +46,14 @@ type UploadParams struct {
 	// Target local path
 	TargetPath string
 
-	UseRegExp bool
-	Flat      bool
-	Recursive bool
-	Explode   bool
-	Override  bool
-	Publish   bool
-	Deb       string
+	UseRegExp     bool
+	Flat          bool
+	Recursive     bool
+	Explode       bool
+	Override      bool
+	Publish       bool
+	ListDownloads bool
```

Comment:
I suggest renaming *ListDownloads* to *ShowInDownloadList*.
Also, let's update the README with this addition.